### DISCUSSION TITLE
Random computers keep joining my groups

### DISCUSSION THREAD

**Discussion by alexfare** [Ylianst/MeshCentral#4930] [Q&A] [answered]
I keep getting random computers joining my Groups. Computer names are usually random numbers & one is named Johns-PC. Last night I was setting up a PC and another one joined, Didn't realize it wasn't my PC and I remoted into it. It was most certainly not my PC and it had 100s of Icons on the desktop, I keep mine clean. They quickly opened task manager and ended it. Anyone else have this issue? Is it safe? Can they get into my computers by joining the group?

**Comment by si458** [comment, marked as answer]
https://ylianst.github.io/MeshCentral/meshcentral/faq/#help-ive-been-hacked-there-are-weird-agents-appearing-in-my-meshcentral-console
EDIT: link might be helpful, maybe check the computer details tab, get the external ip and query the whois for the ip?

**Reply by alexfare** [reply]
Thank you! I tried searching for it but didn't come up with anything. I'm not too familiar with GitHub. All computers connected that I searched came up as Google Cloud IPs & They're all the same that keep joining my groups. Just weird when I connected to the desktop and It looked like a real persons computer., But I was only connected for like 4 seconds and quickly disconnected.

**Reply by si458** [reply]
yes i had the same experience, i uploaded my exe agent to virustotal, spotted it added devices, then connected to the computer, and looked like a real desktop but then quickly disconnected and never again, i think its to trick viruses into thinking its a real computer, shame the IP range gives it away haha
